datarepo-clienttests/src/main/resources/servers/alpha.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "alpha",
+  "description": "alpha env",
+  "datarepoUri": "https://data.alpha.envs-terra.bio/",
+  "samUri": "https://sam.dsde-alpha.broadinstitute.org",
+  "samResourceIdForDatarepo": "terra-datarepo-alpha",
+  "clusterName": "gke_terra-datarepo-alpha_us-central1_jade-master-us-central1",
+  "clusterShortName": "us-central1",
+  "region": "us-central1",
+  "project": "terra-datarepo-alpha",
+  "namespace": "terra-alpha",
+  "containerName": "datarepo-datarepo-api",
+  "deploymentScript": {
+    "name": "ModularHelmChart",
+    "parameters": ["https://raw.githubusercontent.com/broadinstitute/terra-helmfile/master/terra/values/datarepo/live/alpha.yaml?token=AD5MJSFDAS6EQ2VBUHE5WV27KES7Q"]
```

Comment:
Does this deployment script + parameter work? I know you've set skipDeployment=true below, but I don't think it will work if you flip that flag. The alpha helm chart looks like the umbrella variety, not the modular one.

Reply:
Resources will already be deployed with helm once we've reached alpha.

Reply:
I agree - I don't think it would work. I think I'll just remove this definition!

Reply:
Okay, removing it sounds good, thanks!

If we do want to re-deploy between test runs on alpha in the future, then we'd just need a new deployment script that either deploys an umbrella chart (should be very similar to the modular one) or updates the helmfile and lets argocd handle the actual deployment.